frontend/app/views/fragments/form/feedback.scala.html

```diff
@@ -64,12 +64,14 @@ <h2 class="fieldset__headline">About you</h2>
             @fragments.form.errorMessage("Please enter your name")
         </div>
 
+        @* email regex taken from http://www.w3.org/TR/html-markup/input.email.html#input.email.attrs.value.single *@
         <div class="form-field">
             <label class="label" for="email">Email address</label>
             <input name="email"
                    id="email"
                    class="input-text"
                    type="email"
+                   pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$"
```

Comment:
The pattern/regex work is to stop the blank white page when a non email string is added in the email input. 
After a little natter I'll add a //TODO and move this work server-side when I have some downtime. @wpf500 pointed out there is email check stuff we can use in scala hich feels a better place for it.